algoliasearch-common/src/main/java/com/algolia/search/objects/EditType.java

```diff
@@ -0,0 +1,18 @@
+package com.algolia.search.objects;
+
+
+public enum EditType {
```

Comment:
I wouldn't use `enum`. I agree it's better for the DX, but it's more work to maintain and it needs a release of a new version of the API client if we need to add one value. I would stick to `String`

Reply:
Ok! I will drop it.